src/test/lassipatanen/User/Profile.py

```diff
@@ -106,15 +128,16 @@ class ProfileResponse(CamelCaseModel):
         title="Nationality code",
         description="ISO 3166-1 alpha-2 code for nationality",
         example="FI",
+        nullable=True,
     )
     job_titles: Optional[List[str]] = Field(
-        None,
+        ...,
         title="Job titles",
         description="List of job titles",
         example=["Chef", "Programmer"],
     )
     regions: Optional[List[str]] = Field(
```

Comment:
You probably don't want this to be `Optional` if you change the default to `...`, right?

Reply:
probably not. Removing Optional didn't seem to cause any change in Open API spec file tho, but it has now been removed from the python file.